pkg/sfu/downtrack.go

```diff
@@ -741,6 +721,7 @@ func (d *DownTrack) WritePaddingRTP(bytesToSend int, paddingOnMute bool) int {
 		bytesSent += hdr.MarshalSize() + len(payload)
 	}
 
+	// STREAM_ALLOCATOR-TODO: change this to pull this counter from stream allocator so that counter can be update in pacer callback
```

Comment:
Adding this TODO here as we were returning bytes sent when a write failed synchronously. With pacer, cannot get error synchronously. So, this is speculatively returning bytes sent for all padding packets potentially. Few options
1. Keep counter here and stream allocator can pull from here periodically.
2. Post periodically to stream allocator how much padding is sent.
3. Use a wait group or some blocking mechanism to ensure all packets have been processed, count bytes sent in the callback and return that.